gendiff/formatting.py

```diff
@@ -35,3 +35,57 @@ def get_diff_str(dict1: Dict[str, Any], dict2: Dict[str, Any]) -> str:
 
     output_str += "}"
     return output_str.lower()
+
+
+diffs_templates = {
+    "missing": "{indent}- {key}:{space}{value}\n",
+    "added": "{indent}+ {key}:{space}{value}\n",
+    "no_change": "{indent}  {key}:{space}{value}\n",
+    "nested": "{indent}  {key}: {{\n{value}{indent}  }}\n",
+    "missing_nested": "{indent}- {key}: {{\n{value}{indent}  }}\n",
+    "added_nested": "{indent}+ {key}: {{\n{value}{indent}  }}\n",
+}
+
+
+def convert_value(val: Any) -> Any:
+    exceptions_to_convert = {False: "false", True: "true", None: "null"}
+    return exceptions_to_convert.get(val, val) if not isinstance(val, dict) else val
+
+
+def plain_diff(key_type: str, indent: int, key: str, value: Any) -> str:
+    value = convert_value(value)
+    space = "" if value == "" else " "
+    plain_diff_str = diffs_templates[key_type].format(
+        indent=indent * " ", key=key, space=space, value=value
+    )
+
+    return plain_diff_str
+
+
+def nested_diff(
+    key_type: str, indent: int, key: str, value: Any = None, ch_value: Any = None
+) -> str:
+    plain_ante = plain_post = ""
+    if key_type == "changed_nested_1":
+        plain_post = plain_diff(
+            key_type="added",
+            key=key,
+            indent=indent,
+            value=ch_value,
+        )
+        key_type = "missing_nested"
+
+    if key_type == "changed_nested_2":
+        plain_ante = plain_diff(
```

Comment:
`ante`? what does it mean?

Reply:
means before)))

Reply:
`plain_before` & `plain_after` will be easier to read